diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 8947c8a..ca48856 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -136,7 +136,7 @@ endif (VPP_ENABLE_SANITIZE_ADDR)
 ##############################################################################
 # install config
 ##############################################################################
-set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib")
+set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/${VPP_LIBRARY_DIR}")
 set(CMAKE_INSTALL_MESSAGE NEVER)
 
 include_directories (
