--- vpp-19.08.1/src/CMakeLists.txt	2019-09-18 09:36:13.000000000 -0700
+++ vpp-19.08.1/src/CMakeLists.txt	2019-10-02 14:05:04.100570988 -0700
@@ -50,7 +50,7 @@
 		      compiler_flag_no_address_of_packed_member)
 
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
-set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/${CMAKE_INSTALL_LIBDIR})
 
 if (CMAKE_BUILD_TYPE)
   set(CMAKE_C_FLAGS "-g -fPIC ${CMAKE_C_FLAGS}")
@@ -71,7 +71,7 @@
 ##############################################################################
 # install config
 ##############################################################################
-set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib")
+set(CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}")
 set(CMAKE_INSTALL_MESSAGE NEVER)
 
 include_directories (
